src/fcmp_pp/fcmp_pp_rust/.gitignore

```diff
@@ -0,0 +1,4 @@
+# If a developer runs cargo build inside this sub-directory to only work with
+# the Rust side of things, they'll create this target directory which shouldn't
+# be committed
+target
```

Comment:
Is there a way to move the `target` dir to the main `build` directory? Could be helpful if testing out different branches in development?

Reply:
Yep, it's an argument for the target dir with adjusted paths everywhere relevant.